qa/workunits/rados/test.sh

```diff
@@ -78,12 +103,35 @@ ret=0
 if [ $parallel -eq 1 ]; then
 for t in "${!pids[@]}"
 do
-  pid=${pids[$t]}
-  if ! wait $pid
-  then
-    echo "error in $t ($pid)"
-    ret=1
-  fi
+    # Set timeout values
+    max_wait=$timeout
+    waited=0
+    check_interval=10
+    pid=${pids[$t]}
+    echo "Checking Test $t (PID $pid)..."
+    # Check in a loop with timeout
```

Comment:
IIUC, after each test passes, timeout should be updated to 
timeout = timeout - waited

Otherwise according to `timeout=1800  # 30 minutes default value` we would get 30 minutes for **each** test. A timeout for each test is also probably ok but should be noted in the comment above that this is the intent.

Reply:
Yes, well now after Bill's suggestion, we increased the timeout to 5400 (90 mins) for each test.

Will add more descriptive comments at the top so user is well aware that it is **for each test**